src/lib.rs

```diff
@@ -1986,10 +1986,12 @@ impl Connection {
         Ok(read)
     }
 
-    /// Writes a single QUIC packet to be sent to the peer.
+    /// Writes a single QUIC packet to be sent to the peer along with send_info
+    /// like to time to send the packet out.
```

Comment:
This will be available in the public crate docs. I think generally it would help to expand the documentation here a bit. Can you link to the `SendInfo` object rather referring to it as `send_info`?

Reply:
made the changes.